### PR TITLE
Dataumized credits/dreams, added skeletal content package system.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Prep for #41.
- Renamed SSculture to SSlore.
- Broke dreams and credits terms up across several `/decl/content_package_strings` objects.

### CODE REVIEW

File: code/controllers/subsystems/initialization/culture.dm [hunk 1]
```diff
@@ -26,7 +41,66 @@ SUBSYSTEM_DEF(culture)
...
+			credits_crew_names |= manifest.credits_crew_names
+		if(LAZYLEN(manifest.credits_holidays))
+			credits_holidays |= manifest.credits_holidays
+		if(LAZYLEN(manifest.credits_adjectives))
+			credits_adjectives |= manifest.credits_adjectives
+		if(LAZYLEN(manifest.credits_crew_outcomes))
+			credits_crew_outcomes |= manifest.credits_crew_outcomes
+		if(LAZYLEN(manifest.credits_topics))
+			credits_topics |= manifest.credits_topics
+		if(LAZYLEN(manifest.credits_nouns))
```
Comment (PsiOmegaDelta) [inline code/controllers/subsystems/initialization/culture.dm]: Are these lists really LAZY or is `LENGTH()` more appropriate?

Comment (MistakeNot4892) [reply code/controllers/subsystems/initialization/culture.dm]: It doesn't really matter considering `LAZYLEN()` is a macro for `length()`.

File: code/controllers/subsystems/initialization/culture.dm [hunk 2]
```diff
@@ -6,9 +6,24 @@ SUBSYSTEM_DEF(culture)
...
-/datum/controller/subsystem/culture/proc/get_all_entries_tagged_with(var/token)
-	return tagged_info[token]
+	var/list/credits_other =           list("ATTACK! ATTACK! ATTACK!")
+	var/list/credits_adventure_names = list("QUEST", "FORCE", "ADVENTURE")
+	var/list/credits_crew_names =      list("EVERYONE")
+	var/list/credits_holidays =        list("HOLIDAY", "VACATION")
+	var/list/credits_adjectives =      list("SEXY", "ARCANE", "POLITICALLY MOTIVATED")
+	var/list/credits_crew_outcomes =   list("PICKLED", "A VALUABLE HISTORY LESSON", "A BREAK", "HIGH", "TO LIVE", "TO RELIVE THEIR CHILDHOOD")
+	var/list/credits_topics =          list("SACRED GEOMETRY","ABSTRACT MATHEMATICS","LOVE","DRUGS","CRIME","PRODUCTIVITY","LAUNDRY")
+	var/list/credits_nouns =           list("DIGNITY", "SANITY")
```
Comment (PsiOmegaDelta) [inline code/controllers/subsystems/initialization/culture.dm]: Dreams I can certainly see intermingled with culture, but credits seems like more of a stretch

File: code/controllers/subsystems/initialization/culture.dm (hunk 1 above)
Comment (PsiOmegaDelta) [reply code/controllers/subsystems/initialization/culture.dm]: Programmatically it doesn't matter, no, it's more about what expectations one can have about lists that are used along with LAZY-macros.

File: code/controllers/subsystems/initialization/culture.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/controllers/subsystems/initialization/culture.dm]: The sole purpose of the culture datum is to hold lore datums. It seemed as appropriate as anything given that this is all fluff. If you have a preferred subsystem I can shift it over to it.

File: code/controllers/subsystems/initialization/culture.dm [hunk 3]
```diff
@@ -26,7 +41,66 @@ SUBSYSTEM_DEF(culture)
 				tagged_info[culture.category] = list()
 			var/list/tag_list = tagged_info[culture.category]
 			tag_list[culture.name] = culture
+
+
+	for(var/jobtype in subtypesof(/datum/job))
+		var/datum/job/job = jobtype
+		var/title = initial(job.title)
+		if(title)
+			dreams |= "a [title]"
```
Comment (PsiOmegaDelta) [inline code/controllers/subsystems/initialization/culture.dm]: `"\a [title]"` should work here

Comment (MistakeNot4892) [reply code/controllers/subsystems/initialization/culture.dm]: Titles are generally capitalized.

File: code/controllers/subsystems/initialization/culture.dm (hunk 2 above)
Comment (MistakeNot4892) [reply code/controllers/subsystems/initialization/culture.dm]: Renamed SSculture to SSlore.

**Review by PsiOmegaDelta — APPROVED** [review]

File: code/controllers/subsystems/initialization/culture.dm (hunk 3 above)
Comment (PsiOmegaDelta) [reply code/controllers/subsystems/initialization/culture.dm]: `"\an ["\improper [title]"]`

**Pull request merged**